modules/rpc/handler.go

```diff
@@ -26,6 +26,7 @@ import (
 	"go.uber.org/fx"
 	"go.uber.org/fx/internal/fxcontext"
 	"go.uber.org/fx/service"
+	"go.uber.org/fx/uauth"
```

Comment:
Agreed when the term overlaps with standard lib. In this case, it doesn't, and you wouldn't be using 2 auth libs at the same time.